SDK-1204 Updated docs (#686)
box/box-ios-sdk: docs/usage/legal-holds.md

To retrieve information about the items contained in a folder, call
[`client.legalHolds.listForEnterprise(policyName: String, marker: String?, limit: Int?, fields: [String]?)`][get-legal-hold-policies]

> with the ID of the policy.  This method will return an iterator object in the completion you can use to retrieve policies in the enterprise.

Reviewer: ```suggestion
with the ID of the policy.  This method will return an iterator object in the completion, which is used to retrieve policies in the enterprise.
```

To retrieve legal hold policy assignments, call
[`client.legalHolds.listPolicyAssignments(policyId: String, assignToType: String?, assignToId: String?, marker: String?, limit: String?, fields: [String]?)`][get-policy-assignments]

> with the ID of a policy.  This method will return an iterator object in the completion that you can use to retrieve policies assignments for a policy.

Reviewer: ```suggestion
with the ID of a policy.  This method will return an iterator object in the completion, which is used to retrieve policy assignments for a policy.
```

To retrieve all of the non-deleted legal holds for a single legal hold policy, call
[`client.legalHolds.listFileVersionPolicies(policyId: String, marker: String?, limit: String?, fields: [String]?)`][get-file-version-legal-holds]

> with the ID of a policy.  This method will return an iterator object in the completion that you can use to retrieve legal holds for a policy.

Reviewer: ```suggestion
with the ID of a policy.  This method will return an iterator object in the completion, which is used to retrieve legal holds for a policy.
```
